src/taco-dependency-installer/resources/en/resources.json

```diff
@@ -11,6 +11,10 @@
     "_InstallDestination.comment": "Label and value for the installation directory of a dependency. {0} is the path where the dependency will be installed.",
     "DependencyLicense": "License terms: <link>{0}</link>",
     "_DependencyLicense.comment": "Label and value for the url to the license terms of a dependency. {0} is the url.",
+    "DownloadSize": "Download size: {0} MB (approximately)",
```

Comment:
would recommend keeping size information in json file with units. so that we can show based on size, say 12 KB or 23 MB or 1 GB. this information doesn't need to be localized anyways